ui/admin/app/components/form/credential/ssh_private_key/index.hbs

```diff
@@ -117,25 +122,29 @@
         </F.Error>
       {{/if}}
     </Hds::Form::Textarea::Field>
-    <form.input
-      @type='password'
-      @name='private_key_passphrase'
+
+    <Hds::Form::TextInput::Field
       @value={{@model.private_key_passphrase}}
-      @label={{t 'resources.credential.form.private_key_passphrase.label'}}
-      @helperText={{t 'resources.credential.form.private_key_passphrase.help'}}
-      @error={{@model.errors.private_key_passphrase}}
-      @hasVisibilityToggle={{false}}
```

Comment:
Should the new field have this set to false as well? I don't think it was doing anything before?

Reply:
@ZedLi yea I just looked for references in the code base to what this actually does, and we have quite a few instances of this, but still couldn't find the source of it.  I think the `type` argument on `Hds::TextInput::Field` generates this for us.  I will remove it.

Reply:
No, I think we should set it to false as the default is true if you don't set it. I think this particular one was set accidentally on an old rose input but I think we have set it to false for newer HDS password fields. The field is an HDS [one](https://helios.hashicorp.design/components/form/text-input?tab=code#password-1).

@DhariniJeeva might have more insight into this.

Reply:
@ZedLi ahh ok I see what you mean

Reply:
ah yes, it does not do anything with rose components and was accidentally added.

Reply:
@ZedLi @DhariniJeeva incoming commit to fix.  Also I added `data-test-*` attributes for our error messaging.  Let me know what you think.

Reply:
We still need it on the other password field

Reply:
sorry, yeah it doesn't seem to fit here, we can go back to specific error class

Reply:
Actually now I'm not exactly sure if we should keep it false. We [decided](https://hashicorp.slack.com/archives/C0489FTC7SN/p1698425134186789?thread_ts=1698372586.672099&cid=C0489FTC7SN) to do it for just secret type fields and not user passwords but we should check in with design again to see if anything has changed and if we should set it to true now

Reply:
@joshuaogle what do you think?

Reply:
Thinking about this some more, I do think that guidance was for the old fields which worked a little differently. Since hasVisibilityToggle is the default for the password type, I think we can leave it true now :+1:

Reply:
Sorry for the back and forth on this, it's been hard to get consensus on how these should work to give us the best convenience and security, but I think Helios combines the best of all the options.